Java/cp/Shortest_Dist.java

```diff
@@ -0,0 +1,86 @@
+/* Given a list of words and two words word1 and word2, find the shortest distance between those two words from the list.
+ */
+import java.util.*;
+class Shortest_Dist {
+    static List < String > arr;
+    public static void main(String args[]) {
+        int t = 0;
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the list of words " + "\n" + "Words = ");
+        arr = new ArrayList <> ();
+
+        arr.add(sc.nextLine());
+        while (sc.hasNext()) {
+            arr.add(sc.nextLine());
+            if (sc.hasNextInt()) {
+                t = sc.nextInt();
+                break;
+            }
+
+        }
+
+        List < String > wd = new ArrayList(arr);
+        while (t > 0) {
+            String wd1 = sc.next();
```

Comment:
Prompt user to enter the words between which the distance is to be determined.

Reply:
@atarax665 done.